import sys

sys.stderr.write('error')
print 'valid'
